app/src/main/java/com/wootric/androidsdk_demo/AnotherActivity.java

```diff
@@ -0,0 +1,22 @@
+package com.wootric.androidsdk_demo;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentActivity;
+
+public class AnotherActivity extends FragmentActivity {
```

Comment:
Except for leak detection, I don't think there is any value keeping this other activity in the sample app of the SDK (as mentioned in my [PR](https://github.com/Wootric/WootricSDK-Android/pull/98)). Up to you if you would like to keep it.

Reply:
I think I'll keep. It's just a sample app so it doesn't hurt :)